cmd/utils/flags.go

```diff
@@ -947,13 +957,32 @@ func MakeAddress(ks *keystore.KeyStore, account string) (accounts.Account, error
 	return accs[index], nil
 }
 
+// setValidator retrieves the validator address either from the directly specified
```

Comment:
Can we include comments above each of these functions describing the difference between validator, etherbase, and blsbase?

Reply:
Let me know if the added comments are sufficient